serenity-automation-mentoring/src/test/java/serenity/bdd/pages/PetStorePage.java

```diff
@@ -0,0 +1,54 @@
+package serenity.bdd.pages;
+
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import net.serenitybdd.rest.SerenityRest;
+import net.thucydides.core.annotations.Step;
+import org.apache.http.HttpHeaders;
+import serenity.Config;
+import serenity.models.Pet;
+
+public class PetStorePage {
```

Comment:
This is not a Page, use meaningful namings, i.e. `PetStoreRestSteps` or `PetStoreEndpoint`

Reply:
Done